### PR TITLE
Handle json response better

### PR DESCRIPTION
**Pull request by ashwindasr**
Fixes for https://github.com/openshift-eng/art-bot/pull/67

### CODE REVIEW

**Comment by ashwindasr** [comment]
PR for API server: https://github.com/openshift-eng/art-dashboard-server/pull/48

File: artbotlib/pipeline_image_names.py [hunk 1]
```diff
@@ -27,6 +27,10 @@ def process_data(response_data):
         if distgit['brew']['bundle_distgit'] != "None":
```
Comment (locriandev) [inline artbotlib/pipeline_image_names.py:27]:
```suggestion
        if distgit['brew']['bundle_distgit']:
```

Comment (locriandev) [reply artbotlib/pipeline_image_names.py:27]: Same at line 24 (it won't let me suggest the change)

**Review by locriandev — CHANGES_REQUESTED** [review]

File: artbotlib/pipeline_image_names.py (hunk 1 above)
Comment (ashwindasr) [reply artbotlib/pipeline_image_names.py]: I thought I fixed it... hmm not sure what happened. Fixed in https://github.com/openshift-eng/art-bot/pull/72/commits/c606a1ebeaacab9894581dde0b735cb2c1b0da52

**Review by locriandev — APPROVED** [review]

**Pull request merged**